hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/io/erasurecode/rawcoder/TestDecodingValidator.java

```diff
@@ -123,7 +125,12 @@ protected void performTestValidate(int chunkSize) {
     }
 
     // decode
-    backupAndEraseChunks(clonedDataChunks, parityChunks);
+    try {
```

Comment:
use LambdaTestUtils.intercept() for *all* bits of test code which validates expected exceptions

Reply:
The `backupAndEraseChunks` method would only throw exception if the erased data index is out of bound. This is only true for the newly added value set. 

Adding LambdaTestUtils.intercept() fails the tests for other parameterized values since LambdaTestUtils.intercept() @throws AssertionError if the evaluation call didn't raise an exception.